main.py

```diff
@@ -218,15 +217,21 @@ def _switch_colors(self) -> dict:
     # pylint: disable=missing-timeout
     def _send_flow_mods(self, dpid_flows: dict) -> None:
         """Send FlowMods."""
-        for dpid, flows in dpid_flows.items():
-            res = requests.post(
-                self._flow_manager_url % dpid,
-                json={'flows': flows, 'force': True}
-            )
-            if res.status_code // 100 != 2:
-                log.error(f'Flow manager returned an error inserting '
-                          f'flows {flows}. Status code {res.status_code} '
-                          f'on dpid {dpid}')
+        with httpx.Client() as client:
```

Comment:
Aldo, there's even room to remove the request entirely and do via events, since `_send_flow_mods` is already being sent from a threaded context from an event. So, it doesn't benefit much from the request synchronous response here. Similar to how deletion is being done https://github.com/kytos-ng/coloring/blob/master/main.py#L231

Ultimately, to also handle potential OFPT_ERRORs (rare since the flow match/action is pretty simple and in most pipelines config will work), it needs to handle [`"kytos/flow_manager.flow.error"` as mef_eline does](https://github.com/kytos-ng/mef_eline/blob/master/main.py#L1013), but that can be done in another issue.

Reply:
Changed.